terraform-post-packer/backend.tf

```diff
@@ -5,6 +5,6 @@ terraform {
     dynamodb_table = "terraform-state-lock"
     profile        = "cool-terraform-backend"
     region         = "us-east-1"
-    key            = "teamserver-packer/terraform-post-packer.tfstate"
+    key            = "terraform-packer/terraform-post-packer.tfstate"
```

Comment:
I think this should not have been changed from `teamserver-packer/terraform-post-packer.tfstate`.

Reply:
See commit 8fd8aaf.